opencti-platform/opencti-front/src/components/FilterIconButton.tsx

```diff
@@ -80,4 +74,43 @@ const FilterIconButton: FunctionComponent<FilterIconButtonProps> = ({
   );
 };
 
+const FilterIconButton: FunctionComponent<FilterIconButtonProps> = ({
+  availableFilterKeys,
+  filters,
+  handleRemoveFilter,
+  handleSwitchGlobalMode,
+  handleSwitchLocalMode,
+  styleNumber,
+  disabledPossible,
+  redirection,
+  chipColor,
+  helpers,
+  availableRelationFilterTypes,
+}) => {
+  const displayedFilters = filters ? {
+    ...filters,
+    filters:
+      filters.filters.filter(
+        (f) => !availableFilterKeys || availableFilterKeys?.some((k) => f.key === k),
```

Comment:
this will never turn to `[]` as `filters.filters.filter(...)` is always defined

Reply:
done